timelock-server/src/integTest/resources/paxosThreeServers.yml

```diff
@@ -0,0 +1,28 @@
+algorithm:
+  type: paxos
+  paxosDataDir: <TEMP_DATA_DIR>
+
+cluster:
+  localServer: localhost:8080
+  servers:
+    - localhost:8080
+    - localhost:8082
+    - localhost:8084
+
+clients:
+  - test
+  - test2
+  - test3
+  - learner
+  - acceptor
```

Comment:
I originally copypasted the 1-server case, will just leave the one client that's used in the 3-server test.